notes-service/src/main/java/org/exoplatform/wiki/utils/Utils.java

```diff
@@ -684,4 +692,11 @@ public static String html2text(String html) {
     return doc.text();
   }
 
+  public static <S, D> void broadcast(ListenerService listenerService, String eventName, S source, D data) {
+    try {
+      listenerService.broadcast(eventName, source, data);
+    } catch (Exception e) {
+      LOG.error("Error while broadcasting event: {}", eventName, e);
```

Comment:
Use warn log level when the operation will not break the rest of computation. This will be used just for logging and troubleshooting.

```suggestion
      LOG.warn("Error while broadcasting event: {}. Wheras, the operation will continue to proceed", eventName, e);
```

Reply:
done